services/mail/mail-common/src/message.ts

```diff
@@ -247,7 +248,8 @@ async function saveMessageToSpaces (
         modifiedBy,
         MessageType.Message,
         {
-          created: createdDate
+          created: createdDate,
+          edited: createdDate
```

Comment:
Looks extra

Reply:
`await msgClient.createMessage( threadId,  chat.masterTag.Thread,   content,  modifiedBy MessageType.Message, undefined, created)` something like this?